### DISCUSSION TITLE
Trouble in assembling plant mitogenome

### DISCUSSION THREAD

**Discussion by curtainDian** [Kinggerm/GetOrganelle#193] [Q&A]
Hi, Mr. Kinggerm. I adjust my parameters from many aspects (--max-reads  -w --reduce-reads-for-coverage ), but nearly all of the results are incomplete. One of the files of get_org.log.txt is below, could you please offer me some advice ? Thanks !
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/9522790/get_org.log.txt)

**Comment by Kinggerm** [comment]
Please provide a visualized version of the *fastg file (preferred for your data privacy) or the original fastg file

**Reply by curtainDian** [reply]
okay，it’s my visulalized version of the *fastg file.
![image](https://user-images.githubusercontent.com/53113822/189508723-dc5445a5-c64d-4b54-a7fa-0fcf6d04177b.png)

**Reply by Kinggerm** [reply]
First of all, there is no guarantee for the completeness of plant mt using Illumina data. The completeness rate is extremely low because of repeats. I always recommend only working with scaffolds. 

The graph seems to be fragmented with a small number of repeats. For this dataset and a completeness goal, I would 
1. not use `--max-extending-len` for mitogenome extension.
2. use a slightly smaller word size; and/or with less or more reads.
